cnv/cnv_users_guide/cnv-using-the-cli-tools.adoc

```diff
@@ -0,0 +1,21 @@
+[id="cnv-using-the-cli-tools"]
+= Using the CLI tools
+include::modules/cnv-document-attributes.adoc[]
+:context: using-the-cli-tools
+toc::[]
+
+There are two primary CLI tools used for managing resources in the cluster:
+
+* The {ProductName} `virtctl` client, and 
```

Comment:
Do we need the `and` here? I don't usually use `and` in a bullet list, as that what a list is for!